external-resolver/src/main/scala/org/genivi/sota/resolver/vehicle/VehicleDirectives.scala

```diff
@@ -37,7 +40,16 @@ class VehicleDirectives(implicit db: Database, mat: ActorMaterializer, ec: Execu
     pathPrefix("vehicles") {
       get {
         pathEnd {
-          complete(db.run(VehicleDAO.list))
+          parameter('package.as[Package.NameVersion].?) {
+            case Some(nameVersion) =>
+              val packageName   : Package.Name    = Refined(nameVersion.get.split("-").head)
```

Comment:
I don't think it is a good idea to parse name and version. Name also can contain `-`. What is a reason to pass it as a single string anyway?